cli/localSqsPuller.ts

```diff
@@ -13,12 +13,16 @@ import fetch from 'node-fetch';
  */
 
 // This hardcode should be in sync with values in local_start.sh
```

Comment:
Looks like the comment should be deleted as well

Reply:
`sqsQueueUrl` is still hardcoded, I'll get around it next time we touch configuration-related code